apps/src/gamelab/GameLabSprite.js

```diff
@@ -105,6 +105,36 @@ module.exports.createSprite = function (x, y, width, height) {
     }
   });
 
+  Object.defineProperty(s, 'width', {
```

Comment:
For the sake of future-our sanity, let's make a comment here about how/why we are overriding the original p5.play `Sprite.width` and what's different in this version.